docs/model-cards/marin_8b_base.md

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
kebab case?

Reply:
what? this is yaml metadata for markdown

Reply:
Sorry, I meant this comment on the filename.